recipes/traffic/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set version = "2.6" %}
+
+package:
+  name: traffic
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/t/traffic/traffic-{{ version }}.tar.gz
+  sha256: c9bdcce90139d8334ca4662d53fa8f457c52f3ca4f93f5d833ce1b31831f390d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.18
+    - scipy
+    - matplotlib-base
+    - pandas >=1.0.0
+    - pyproj >=2.0
+    - cartes >=0.2
+    - Shapely
```

Conda package names are always lowercase
```suggestion
    - shapely
```